PurchaseController/Classes/ReceiptValidation/LocalReceiptValidatorImplementation.swift

```diff
@@ -436,11 +436,17 @@ fileprivate struct ReceiptParser {
             return nil
         }
         
-        let integer = c2i_ASN1_INTEGER(nil, &intPointer, intLength)
-        let result = ASN1_INTEGER_get(integer)
-        ASN1_INTEGER_free(integer)
+//         let integer = c2i_ASN1_INTEGER(nil, &intPointer, intLength)
```

Comment:
Please remove unused commented code

Reply:
Please check new updated PR, I removed the commented code